xbmc/TextureCacheJob.cpp

```diff
@@ -32,7 +32,7 @@
 #include "FileItem.h"
 #include "music/MusicThumbLoader.h"
 #include "music/tags/MusicInfoTag.h"
-#if defined(HAS_OMXPLAYER)
+#if defined(TARGET_RASPBERRY_PI)
```

Comment:
No they don't because you need them all for the rpi to even build. So it is stupid to have multiple defines even though they all reach the same goal.

Some of the defines are set currently just for the sake of allowing the rbpi platform to build.
See, https://github.com/xbmc/xbmc/blob/master/cmake/scripts/rbpi/ArchSetup.cmake#L33...%23L35
And, https://github.com/xbmc/xbmc/blob/master/cmake/scripts/rbpi/ArchSetup.cmake#L2

This is confusing and dumb. It is like this because of legacy code and needs to be cleaned up.